src/sqlx_store.rs

```diff
@@ -58,9 +95,7 @@ macro_rules! impl_user_store {
             type User = User;
 
             async fn load_user(&self, user_id: &str) -> crate::Result<Option<Self::User>> {
```

Comment:
@maxcountryman 
I've been wondering ...
the way I structured User record in my app means a PK bigint generated as identity, which requires casting in Postgres in `where` clause.
What if we typed return value of `get_id` as something that is bindable in queries, but not necessarily `str`?
I am not versed enough in Sqlx to suggest something (I don't even know if there's a trait like that right now), but this could provide more versatility.
Of course there's a case to be made that it's unnecessary as soon as someone decides to jot their own query down, because then anything goes in the query itself. I don't know 😅

Reply:
I'm open to this. I do think it would require a little more investigation into what SQLx offers to support here.